test/integration/projects_test.rb

```diff
@@ -0,0 +1,83 @@
+require "test_helper"
+
+class ProjectsTest < ActionDispatch::IntegrationTest
+  test "create" do
+    before = Project.all.size
+    Dir.mktmpdir do |tmpdir|
+      params = {
+        project: {
+          name: "test project one",
+          description: "test project one description",
+          directory: tmpdir
+        }
+      }
+      post projects_path, params: params
+      follow_redirect!
+      assert_response :success
+
+      get projects_path
+      assert_response :success
+
+      after = Project.all.size
+      assert after == before + 1
+    end
+  end
+
+  test "deleting_a_project" do
+    before = Project.all.size
+    Dir.mktmpdir do |tmpdir|
+      params = {
+        project: {
+          name: "test project one",
+          description: "test project one description",
+          directory: tmpdir
+        }
+      }
+      post projects_path, params: params
+
+      id = @response.location.to_s.split("/").last
+
+      delete project_path(id)
+      follow_redirect!
+      assert_response :success
+
+      after = Project.all.size
+      assert after = before -1
```

Comment:
This test uses assignment `=` not equivlance. You should use `assert_equal foo, bar` instead of `assert foo == bar`